test/test_manifest.json

```diff
@@ -3327,6 +3339,12 @@
       "rounds": 1,
       "type": "eq"
     },
+    {  "id": "bug1425312",
```

Comment:
This entry needs `"link": true,` added, to fix the following from the test logs:
```
WARNING: Unable to open file for reading "Error: ENOENT: no such file or directory, open 'pdfs/bug1425312.pdf'".
Unable to verify the checksum for the files that are used for testing.
Please re-download the files, or adjust the MD5 checksum in the manifest for the files listed above.
```

Also, it'd probably suffice to only test the second page here.